### PR TITLE
Updating missing dependency validation error codes

### PR DESCRIPTION
**Pull request by Marraz**
(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@Marraz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Marraz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/modeling/validate-code-with-layer-diagrams.md [hunk 1]
```diff
@@ -223,6 +223,8 @@ The following section describes the syntax that is used in these errors, explain
...
 | DV1003: **Disallowed namespace name** | This issue is reported on a code element associated with a layer which "Disallowed Namespace Names" property contains the namespace in which this code element is defined. This is a naming constraint violation. Note that the "Disallowed namespace name" property is defined as a Semi-colon separated list of namespaces in which code elements associated with this Layer should not be defined. |
+| DV2001: **Layer Diagram Presence** | This issue is reported on a project that do not include a dependency diagram file but refere the dependency validation analyzers. If Dependency Validation is not been use, you can remove "Microsoft.DependencyValidation.Analyzers" directly from Solution Explorer or supress this warning. To add a dependency diagram see [Create dependency diagrams from your code](../modeling/create-layer-diagrams-from-your-code.md). |
```
Comment (JoshuaPartlow) [inline docs/modeling/validate-code-with-layer-diagrams.md]: Suggest edit of text to: 

This issue is reported on a project that does not include a dependency diagram file, but refers to the dependency validation analyzers. If Dependency Validation has not been used, you can remove "Microsoft.DependencyValidation.Analyzers" directly from Solution Explorer or suppress this warning. To add a dependency diagram see [Create dependency diagrams from your code](../modeling/create-layer-diagrams-from-your-code.md)

**Comment by PRMerger6** [comment]
@Marraz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by JoshuaPartlow** [comment]
Reviewed the changes and they look fine to me. #sign-off

**Comment by PRMerger7** [comment]
@JoshuaPartlow: I'm sorry - only the author of this article, @jillre, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by jborsecnik** [comment]
@jillre, ok to merge?

**Comment by jillre** [comment]
Yes, ok to merge. #sign-off

**Pull request merged**